elastic/service.go

```diff
@@ -496,6 +506,10 @@ func (s *service) additionalWhere(dataSourceType, sortField string) (string, err
 		switch sortField {
 		case "github_pull_request_prs_created":
 			return `and \"id\" is not null and \"pull_request\" = true`, nil
+		case "github_pull_request_prs_open":
+			return `and \"state\" = 'open'`, nil
```

Comment:
I would keed `and \"id\" is not null` part

Reply:
on it